bench/crow/optimized/example.cpp.ll

```diff
@@ -69977,61 +69977,59 @@ define linkonce_odr dso_local void @_ZN4asio2ip6detail8endpointC2ERKNS0_7address
           catch ptr null
   call void @_ZNSt8bad_castD2Ev(ptr noundef nonnull align 8 dereferenceable(8) %5) #36, !noalias !1268
   call void @llvm.lifetime.end.p0(i64 8, ptr nonnull %5) #36, !noalias !1268
-  br label %41
+  br label %40
 
 18:                                               ; preds = %10
   %19 = getelementptr inbounds nuw i8, ptr %1, i64 4
   %20 = load i32, ptr %19, align 4, !tbaa !97, !noalias !1268
-  br label %37
+  br label %36
 
 21:                                               ; preds = %3
   store i16 10, ptr %0, align 4, !tbaa !14
   store i16 %8, ptr %9, align 2, !tbaa !14
-  %22 = getelementptr inbounds nuw i8, ptr %0, i64 4
-  store i32 0, ptr %22, align 4, !tbaa !14
```

Comment:
Thought this was being miscompiled, but `%0` is unconditionally memset'd first, so this should be fine?